tasks/cephfs/fuse_mount.py

```diff
@@ -100,11 +100,16 @@ def list_connections():
         self.fuse_daemon = proc
 
         # Wait for the connection reference to appear in /sys
+        mount_wait = self.client_config.get('mount_wait', 0)
+        if mount_wait > 0:
+            log.info("Fuse mount waits {0} seconds before checking /sys/")
+            time.sleep(mount_wait)            
```

Comment:
Hmm, why do we sleep first and then start checking?  Why not use a 180s timeout instead of a 60s wait and a 120s timeout?

Other than that this looks good.